README.md

```
@@ -0,0 +1,104 @@
+| `pnpm android` | Run on Android emulator |
+| `pnpm build` | Bundle server for production (esbuild) |
+| `pnpm start` | Run production server |
+| `pnpm check` | TypeScript type checking |
+| `pnpm lint` | Run ESLint |
+| `pnpm format` | Format code with Prettier |
+| `pnpm test` | Run tests (Vitest) |
+| `pnpm db:push` | Generate and apply database migrations |
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)